src/SourceTerms/LightBulb/CPU_Src_Lightbulb.cpp

```diff
@@ -208,6 +218,18 @@ static void Src_Lightbulb( real fluid[], const real B[],
 #  endif
 
 
+// 4. calculate temperature for initial guess and store to fluid[TEMP_IG]
```

Comment:
`update temperature initial guess` as that in `*_Src_Deleptonization.cpp`

Reply:
I updated the comment.